src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir-Book", sans-serif;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.search {
+  position: relative;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
```

Comment:
Why margins? This image has position: absolute and can be aligned by `top`, `bottom`, `right`, `left` properties